### PR TITLE
Add unit tests for music album and genre class methods

### PR DESCRIPTION
**Pull request by Lembani**
### This PR contains the following implementation:

> Highlights:

- Unit tests for all implemented methods.
- Modified the Item class with associations to all other classes.
- Other minor fixes.

### CODE REVIEW

File: spec/music_album_spec.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  end
+
+  it 'should return a publish date' do
+    expect(@music_album.publish_date).to eq(Date.parse('2010-10-25'))
+  end
+
+  it 'should return false if music album is not on spotify' do
+    expect(@music_album.on_spotify).to eq(false)
+  end
+end
```
Comment (christianbiring1) [inline spec/music_album_spec.rb]: Kindly implement the test for the method `can_be_archived` in the class music_album.

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hey @Lembani,
Good job so far for implementing the suggested changes and the test for the implemented classes.
However, there is still some work left. I highly recommend implementing them to take the project to the next level. :rocket:

File: spec/music_album_spec.rb (hunk 1 above)
Comment (Lembani) [reply spec/music_album_spec.rb]: It's done! 👍🏾

**Review by christianbiring1 — APPROVED** [review]
Great job!
All tests are passing!.
you can go on to the next task.

**Pull request merged**